src/content/reference/faqs.mdx

```diff
@@ -246,3 +246,13 @@ You can only create `Roles` restricted to the endpoints above. `RoleBindings` ca
 This is related to how syncthing interacts with `tsc`. Syncthing creates a temporary file and replaces the original file with the new one.
 
 To solve the problem you just add the flag `--synchronousWatchDirectory` to your `tsc` command.
+
+## I cannot connect to the Kubernetes cluster using the kubeconfig file generated by Okteto
+
+Kubeconfig file generated by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to get credentials for your Kubernetes clusters via Okteto backend. It is a common pattern used by many Kubernetes providers, such as Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS), or Amazon Elastic Kubernetes Service (EKS) to connect to Kubernetes clusters in a more secure way.
+
+For this reason, it is required to have Okteto CLI installed in your PATH and have it connected to your Okteto instance. In order to install the CLI in your path you can follow [our documentation](https://www.okteto.com/docs/getting-started/#installing-okteto-cli).
```

Comment:
I like Okteto instance as I think it's easier to visualize that Okteto can be just a part of a cluster. Anyways, I think both are synonyms and they could be exchanged to avoid to be repetitive in places where we mention it several times. 

Doing a quick search in the docs, we use more `Okteto instance` (758 hits) vs Okteto cluster (76 hits).